roles/ffh.mesh_wireguard/templates/wireguard.conf.j2

```diff
@@ -0,0 +1,6 @@
+[Match]
+Name={{ mesh_wireguard_iface }}-{{ '%02d' % domain.id | int }}
+
+[Network]
+Address = fe80::1/64
+VXLAN={{ vxlan_iface }}-{{ '%02d' % domain.id | int }}
```

Comment:
What does this option do? We don't want to couple the WireGuard interface with VXLAN. We only want the VXLAN UDP socket to use the WireGuard as output to reach it's VX peer in normal routing terms.

Reply:
not sure why I did that.

Reply:
Without it the vxlan devices do not appear and the services add-vx-to-batX do therefore never start.
I'm not sure, what should start my xlan interfaces; gotta look into it.

Reply:
I'm not yet satisfied.
Though I got rid of the VXLAN entry and established vx interaces through the bridge-hack we spoke of,
there is not much happening on the outside.

None of the devices can connect to batman.

Reply:
Gotta rollback.

Reply:
done.